app/src/main/java/com/yeonae/chamelezone/view/login/LoginFragment.kt

```diff
@@ -1,19 +1,28 @@
 package com.yeonae.chamelezone.view.login
 
 import android.os.Bundle
+import android.util.Log
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import android.widget.Toast
 import androidx.fragment.app.Fragment
+import com.google.gson.JsonObject
 import com.yeonae.chamelezone.AlertDialogFragment
 import com.yeonae.chamelezone.R
+import com.yeonae.chamelezone.network.api.RetrofitConnection
+import com.yeonae.chamelezone.network.model.MemberResponse
+import com.yeonae.chamelezone.network.room.entity.User
+import com.yeonae.chamelezone.network.room.database.UserDatabase
 import kotlinx.android.synthetic.main.fragment_login.*
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
 
 
 class LoginFragment : Fragment() {
-    private val testEmail = "heimish_08@naver.com"
-    private val testPassword = "1234"
+    private val retrofitConnection = RetrofitConnection
+    private var userDatabase : UserDatabase? = null
```

Comment:
```suggestion
    private val userDatabase by lazy{
         UserDatabase.getInstance(requireContext()) 
     }
```
여기서 그냥 만들어줘도 될 것 같아요. 
늦게 만들어주고 싶다면 null보다는 lateinit이 적합 할 것 같구요

Reply:
수정했습니다